contracts/SportMarkets/Parlay/ParlayMarket.sol

```diff
@@ -102,6 +102,31 @@ contract ParlayMarket is OwnedWithInit {
         }
     }
 
+    function hasMarketLostButHasExercisableWinningPositions()
```

Comment:
as this doesnt exist in currently created Parlays, need to make sure it wont cause issue